src/main/kotlin/org/rust/ide/typing/paste/RsImportCopyPasteProcessor.kt

```diff
@@ -219,5 +273,11 @@ private fun createFqnMap(file: RsFile, range: TextRange): ImportMap {
 
 private fun gatherElements(file: RsFile, range: TextRange): List<PsiElement> =
     CollectHighlightsUtil.getElementsInRange(file, range.startOffset, range.endOffset)
+        .filter { elem -> elem !is PsiFile }
```

Comment:
Do you have an example when it is needed?

Reply:
I don't know why, but `gatherElements` started returning the whole file as the last element in the range (possibly since `223` platform?). This broke the visitor functionality, because the code then visited the whole file.